command/issue.go

```diff
@@ -46,7 +46,7 @@ var issueCmd = &cobra.Command{
 	Long: `Work with GitHub issues.
 
 An issue can be supplied as argument in any of the following formats:
-- by number, e.g. "123"; or
+- by number, e.g. "123" (many shells treat # as the start of a comment); or
```

Comment:
Since we don't want to actually document the `#123` syntax being accepted (we don't want to confuse bash or fish users who might be tempted to try that syntax), could you undo the changes to documentation in your PR? All other changes are fine. Thanks for writing tests for this! 💎

Reply:
This response is intentionally not mentioning the #<num> as a supported option.

Reply:
Removed documentation hints as requested.